assets/js/util/tracking/index.test.js

```diff
@@ -78,18 +78,24 @@ describe( 'trackEvent', () => {
 
 		trackEvent( 'category', 'name', 'label', 'value' );
 
-		expect( push ).toHaveBeenCalledWith( [
-			'event',
-			'name',
-			{
-				send_to: config.trackingID,
-				event_category: 'category',
-				event_label: 'label',
-				event_value: 'value',
-				dimension1: 'https://www.example.com',
-				dimension2: 'true',
-				dimension3: config.userIDHash,
-			},
-		] );
+		function makeArguments() {
```

Comment:
This might be a nice thing to throw into the testing utils file: https://github.com/google/site-kit-wp/blob/develop/tests/js/utils.js

I know it's niche, but still 😄

Reply:
Yeah, I think for now it's part of the documentation to have it here.

If we need this anywhere else/again then I'd be all for moving it to utils.